### ISSUE TITLE
CAPT (Cluster API for Tinkerbell)

### ISSUE THREAD

**Issue by mrmrcoleman** [tinkerbell/tinkerbell.org#64]
Mark will work with Avi/Gianluca and Graham to figure out an initial design doc for how this could work.

**Comment by mrmrcoleman** [comment]
This should be done in parallel with the Secrets epic.

**Comment by DailyAlice** [comment]
Is this sort of a duplicate issue to https://github.com/tinkerbell/tink/issues/195? This isn't really a documentation or website issue.

Well, yet anyway! If we do want to have an API and API docs then we can probably either re-open or track in a new issue.

Closing!

**Comment by mrmrcoleman** [comment]
This was another roadmap item, again it needs a new home!

**Comment by thebsdbox** [comment]
I believe @detiber has already done a gap analysis on this so should be able to open an issue where it makes the most sense and we can track it from there.

**Comment by detiber** [comment]
Any objections if I create a new repository under the tinkerbell org where we can track these types of issues, start putting together design docs, and eventually start on the implementation?

**Comment by thebsdbox** [comment]
Sounds *excellent* to me @detiber
